assets/i18n/strings_zh-Hant_TW.i18n.json

```diff
@@ -209,14 +282,11 @@
     },
     "quickSaveNotice": {
       "title": "@:general.quickSave",
-      "content": "自動接受所有檔案傳輸請求。請注意，這會讓此網路中的所有人都可以向你傳送檔案。"
+      "content": "自動接受所有檔案傳輸請求。請注意，這會讓此網路中的所有人都可以直接向您傳送檔案。"
     }
   },
   "tray": {
     "open": "@:general.open",
     "close": "退出 LocalSend"
-  },
-  "aliasGenerator": {
```

Comment:
It is already inheriting from the English version, so don't need to copy the whole list